contrib/man/host-metering.1

```diff
@@ -21,6 +21,34 @@ Default is /etc/host-metering.conf
 
 .SH "ENVIRONMENT VARIABLES"
 .PP
+
+\fBHTTP_PROXY\fR
+Represents the value of the HTTP_PROXY or
+http_proxy environment variable. It will be used as the proxy
+URL for HTTP requests unless overridden by NO_PROXY.
+
+\fBHTTPS_PROXY\fR
+Represents the HTTPS_PROXY or https_proxy
+environment variable. It will be used as the proxy URL for
+HTTPS requests unless overridden by NoProxy.
```

Comment:
`NoProxy` should by capitalized as you did it in `HTTP_PROXY`

Reply:
Fixed :)